src/loop_yml.py

```diff
@@ -0,0 +1,17 @@
+import os
+import re
+
+n = 0
+
+for filename in os.listdir(r'C:\Users\nathi_000\Desktop\Python Files\NLP Project\nlp-web-scrapping\src\experiment_configs'):
```

Comment:
Folder names can not be hard coded, as every machine has different folder structure

Reply:
Fair point

I'll alter that